website/common/script/libs/getItemInfo.js

```diff
@@ -177,6 +177,25 @@ module.exports = function getItemInfo (user, type, item, officialPinnedItems, la
         pinType: 'seasonalSpell',
       };
       break;
+    case 'debuffPotion':
+      itemInfo = {
```

Comment:
Sorry but I'm not super familiar with this code, where is this taking the values for `item.mana `, item.cast, item.immediateUse... ? Because here https://github.com/HabitRPG/habitica/pull/11353/files#diff-d0d5d38815f41cc5237729df242e5ea1R17 they don't seem to exist

Reply:
The main idea here that getItemInfo using get function of lodash with content and path with type params, in content we have spells.js file, where we have all this info in special key https://github.com/HabitRPG/habitica/blob/develop/website/common/script/content/spells.js#L312

I tested it locally through vue dev tools and I can show that all of this data we have on client side 
![image](https://user-images.githubusercontent.com/12300576/64542384-26884f80-d32c-11e9-9507-2a9c2e30a812.png)

Reply:
Thats how we convert path and type of item to content especially for opaquePotion 
![image](https://user-images.githubusercontent.com/12300576/64608610-bc2aea00-d3d3-11e9-9c7c-74f90e67d283.png)
@paglias